### PR TITLE
Fix tests to work with newest version of pytype

### PR DESCRIPTION
**Pull request by matthiaskramm**
We removed some flags. In particular, the typeshed location is now passed via the environment.

### CODE REVIEW

File: tests/pytype_test.py [hunk 1]
```diff
@@ -60,19 +60,23 @@ def load_blacklist():
...
+    def __init__(self, args, dry_run=False, env=None):
         self.args = args
-
-        self.dry_run = dry_run
         self.results = None
 
         if dry_run:
             self.results = (0, '', '')
         else:
+            if env:
```
Comment (gvanrossum) [inline tests/pytype_test.py]: Maybe `if env is None`?

File: tests/pytype_test.py [hunk 2]
```diff
@@ -122,11 +126,10 @@ def pytype_test(args):
             if f in pytype_run:
                 test_run = BinaryRun(
                     ['pytype',
-                     '--typeshed-location=%s' % os.getcwd(),
                      '--module-name=%s' % _get_module_name(f),
-                     '--convert-to-pickle=%s' % os.devnull,
-                     f],
-                    dry_run=args.dry_run)
+                     '--parse-pyi', f],
```
Comment (gvanrossum) [inline tests/pytype_test.py]: I'd keep `f` on a line by itself (like it was -- you made it "hide" between a much longer flag).

Comment (matthiaskramm) [reply tests/pytype_test.py]: Done.

**Comment by matthiaskramm** [comment]
Thanks for the review!

**Pull request merged**